libraries/I2S/keywords.txt

```diff
@@ -16,11 +16,14 @@ end	KEYWORD2
 
 setBCLK	KEYWORD2
 setDATA	KEYWORD2
+setMCLK	KEYWORD2 
 setBitsPerSample	KEYWORD2
 setFrequency	KEYWORD2
 setBuffers	KEYWORD2
 setLSBJFormat	KEYWORD2
 swapClocks	KEYWORD2
+setMCLKmult  KEYWORD2
```

Comment:
`keywords.txt` is very fussy and needs to have actual `tab` characters, not spaces, between the word and the `KEYWORD/xxx` type or it will ignore it.  I've been bitten by this myself more than once!